browser/src/DatasetSelector.tsx

```diff
@@ -591,15 +591,15 @@ const DatasetSelector = withRouter(({ datasetOptions, history, selectedDataset }
   if (includeStructuralVariants || includeCopyNumberVariants) {
     const topLevelStructuralVariantDataset = hasStructuralVariants(selectedDataset)
       ? selectedDataset
-      : 'gnomad_sv_r2_1'
+      : 'gnomad_sv_r4'
```

Comment:
Now that there are two different datasets that have structural datasets, we'll need additional logic to determine the topLevelStructuralVariantDataset.

As it stands at the moment, this correctly updates the default SV dataset for v4 (and v3, which makes sense imo) to the v4 SNVs. It also has the unintended side effect of updating the default dataset for v2 to v4 SNVs. We probably want to keep v2's default SV dataset as `gnomad_sv_r2_1`

- ![Screenshot 2023-11-06 at 07 27 09](https://github.com/broadinstitute/gnomad-browser/assets/59549713/781025aa-7a43-40fa-8aa6-52f2dc24402a)